js/screens/SearchResult/SearchResultScreen.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Text, Button } from 'react-native';
+
+import SearchResult from './SearchResult';
+
+import {
+  getListingsByLocation,
+  unMarshallResult,
+  sortListingsByDateDesc
+} from '../../config/helpers';
+
+import { colors } from '../../config/styles';
+
+export default class SearchResultScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { listings: [], loading: true };
+  }
+  static propTypes = {
+    navigation: PropTypes.object
```

Comment:
Please add default prop.

Also, use either ES6 or ES7 for your components, don't use both.